window/src/os/windows/keycodes.rs

```diff
@@ -101,110 +100,110 @@ fn build_map() -> HashMap<WPARAM, PhysKeyCode> {
         (0x34, PhysKeyCode::K4),
         (0x36, PhysKeyCode::K6),
         (0x35, PhysKeyCode::K5),
-        (VK_OEM_PLUS, PhysKeyCode::Equal),
+        (VK_OEM_PLUS.0, PhysKeyCode::Equal),
```

Comment:
Not sure what the answer is, but the `.0` here is ugly and it would be great to eliminate it if possible.